src/pages/guides/develop/coi.md

```diff
@@ -0,0 +1,143 @@
+# Cross-origin isolation handling
+
+How to ensure your add-ons work properly with the cross-origin isolation headers enforced by Adobe Express.
+
+## Overview
+
+Adobe Express will soon be enforcing cross-origin isolation on the associated domains (i.e., “new.express.adobe.com”) for Chromium-based browsers (including Chrome, Microsoft Edge, Opera, and others). This *may* impact your add-on due to stricter rules enforced by the browser. You’ll want to ensure that any add-ons you’ve developed or are developing now work in this new environment.
```

Comment:
I wonder if we should reframe this to make it future proof. For example, instead of 'Adobe Express will soon be enforcing cross-origin isolation' we can say 'Adobe Express enforces cross-origin isolation'. That way we wont have to update this doc once again.

Reply:
+1 to the above. Otherwise, can we add a tentative date here to make it clear when the devs should start doing these.

Reply:
We don't have a date just yet. It depends on how quickly we're able to test existing add-ons, so it's likely a couple of months away.

I don't want to mislead new developers into expecting that these headers are being sent (since they'll end up assuming their add-on works when it might), so I think we'll have to come back and revisit when the headers start to be enforced.

Reply:
Thinking about it more, we can drop in our current estimate, but indicate we'll update when we know more.

Something like:

> We expect the enforcement of cross-origin isolation headers to begin around the end of 2024. We'll update you the moment we have a more certain date.

Reply:
@kerrishotts I added the estimate into [an info block](https://developer-stage.adobe.com/express/add-ons/docs/guides/develop/coi/#overview) but let me know if you think it should be marked as a warning block instead.